react/bingo/src/bingo/BingoApp.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import BingoGame from './Game'
+import './BingoApp.css';
+
+class BingoApp extends React.Component {
+
+  constructor (props){
+    super(props);
+    this.state = {
+      challengeWord : '',
+      isGameOn : false
+    }
+  }
+
+  handleSubmit(){
+    this.setState ({
+      challengeWord : this.state.challengeWord,
+      isGameOn : true
+    });
+    playBingo(this.state.challengeWord);
```

Comment:
No need to call a function returning a component to trigger a render, all of the rendering happens through 
1) the initial render 2) when state or props changes